docs/source/create/avatars/avatar-standards.md

```diff
@@ -166,11 +166,9 @@ Here is an example of modifying your blendshapes in your FST file:
 
 **ReadyPlayerMe Blendshape**
 
-ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically using a default mapping as follows:
+ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically using a default mapping
```

Comment:
It may be unclear if you don't know about RPM avatars, perhaps explain a bit more?

The default blendshapes used in [ReadyPlayerMe](https://readyplayer.me/) avatars are converted into Vircadia blendshapes automatically using a default mapping